lib/dotcom/trip_plan/alerts.ex

```diff
@@ -8,14 +8,40 @@ defmodule Dotcom.TripPlan.Alerts do
   * at the times they'll be travelling
   """
 
-  alias Alerts.Alert
-  alias Alerts.InformedEntity, as: IE
+  alias Alerts.{Alert, InformedEntity}
   alias Dotcom.TripPlan.{Itinerary, Leg, TransitDetail}
 
+  def by_mode_and_stops(alerts, leg) do
```

Comment:
📓 I just moved this from the bottom of the file since it was after private defs.